interbtc-spec/docs/source/spec/security.rst

```diff
@@ -36,20 +36,14 @@ The frequency of the oracle updates is defined in the Oracle module.
 
 **Error code:** ``ORACLE_OFFLINE``
 
-BTC-Relay Offline
```

Comment:
Note: will need to update the state machine figure.

Reply:
Done: https://lucid.app/lucidchart/9c260a6f-cfd2-42f4-96d5-3e4f95717b98/edit?viewport_loc=-141%2C-1114%2C1646%2C1321%2CBQcUKJSmjDB8